packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -44,6 +45,9 @@ const useDraftOutbound = () => {
 
 export const DetailView: FC = () => {
   const { draft, onChangeSortBy, save, sortBy } = useDraftOutbound();
+  const { prefetchListByName } = useItemsList({
+    initialSortBy: { key: 'name' },
+  });
```

Comment:
this was more of a little experiment than being useful..

basically prefetching the 'next' item  list (for the autocomplete) when you click ok and next